index.html

```diff
@@ -227,35 +254,34 @@ <h1 class="about-title">About me</h1>
     <!-- Contact Form Start -->
 
 
-    <footer>
+    <footer id="footer">
 
         <div id="maincontact">
             <div class="contact">
-                <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in
                     touch.</h2>
             </div>
             <form action="https://formspree.io/f/mrgjwpjk" method="post" class="form-contact">
                 <div id="form-contact">
                     <input type="text" class="name" maxlength="30" required name="name" placeholder="Full name">
-
                     <input type="email" class="email" name="email" required placeholder="Email Address">
                     <textarea class="txt-area" maxlength="500" name="txt-area" required id="txt-area" cols="30" rows="5"
                         placeholder="Enter text here"></textarea>
-                </div>
-                <button class="form" type="submit"> Get in touch</button>
-                <div>
-                    <div id="c-line">
-                        <div></div>
-                    </div>
-                    <div id="s-icons">
-                        <img src="/images/social-media-icon.png" alt="">
-                    </div>
-                    <div class="l-line">
-                        <img src="/images/light.png" alt="">
-                    </div>
+                    <button class="form" type="submit"> Get in touch</button>
                 </div>
             </form>
         </div>
+        <div class="footer">
+            <div id="c-line">
+
+            </div>
```

Comment:
- [ ] Your footer looks great 👏. It is important to note that you should not use empty containers in your HTML. Because they make it hard for someone to understand and they make your code unclean. Try to find the appropriate semantic tag of what you want to achieve. Like in this case, I can see that you wanted to put a horizontal line, 💡**HINT**: There is an HTML tag `<hr>` which is a horizontal line to separate sections so you can use it instead 😃.